inference-engine/src/mkldnn_plugin/nodes/mkldnn_generic_node.cpp

```diff
@@ -101,7 +101,7 @@ bool MKLDNNGenericNode::created(const MKLDNNExtensionManager::Ptr &extMgr) {
         }
         if (impls.empty()) {
             extFactory = extMgr->CreateExtensionFactory(getCnnLayer());
-            extShapeInference = extMgr->CreateReshaper(getCnnLayer());
+            extShapeInference = nullptr;
```

Comment:
Can we remove `extShapeInference`?

Reply:
removed after discussion with @dmitry-gorokhov 

Added a comment below to use ngraph based shape inference if needed